resources/dicts/patrols/beach/med/greenleaf.json

```diff
@@ -2719,6 +2894,71 @@
             }
         ]
     },
+    {
+        "patrol_id": "bch_med_greenleaf_juniper_lorestoryhealerlocked1",
+        "biome": ["beach"],
+        "season": ["greenleaf"],
+        "types": ["herb_gathering"],
+        "tags": [],
+        "patrol_art": "gen_bord_story",
+        "min_cats": 3,
+        "max_cats": 6,
+        "min_max_status": {
+            "healer cats": [1, 6]
+        },
+        "weight": 20,
+        "pl_skill_constraint": ["LORE,1", "STORY,1", "HEALER,1"],
+        "intro_text": "There's an old saying, about searching for juniper.",
```

Comment:
You might want to include the cat telling the story within this text.  After all, we want players to be able to tell that the story/lore cat is the one causing this patrol :P so perhaps "As the patrol sets out, p_l remembers an old saying about searching for juniper."

Reply:
Fixed!